pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,226 @@
+<div class="modal js-rubric-settings-modal rubric-settings-modal-<%= type %>" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-xl border-info" role="document">
+    <form name="rubric-settings-<%= type %>" method="POST" class="needs-validation">
+      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+      <input type="hidden" name="__action" value="modify_rubric_settings">
+      <input type="hidden" name="rubric_type" value="<%= type %>">
+      <input type="hidden" name="modified_at" value="<%= rubric?.modified_at %>">
+      
+      <div class="modal-content">
+        <div class="modal-header bg-info text-light">
+          <h5 class="modal-title">Rubric settings: <%= type %> grading</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <div class="alert alert-danger js-settings-error-alert d-none" role="alert"></div>
+          <div class="form-check">
+            <label class="form-check-label">
+              <input class="form-check-input" name="use_rubrics" type="checkbox" value="true" <% if (rubric) { %>checked<% } %>>
+              Use rubrics
+            </label>
+          </div>
+          <div class="js-rubric-settings-info">
+            <hr/>
+            <div class="form-row">
+              <div class="col-4">
+                <div class="form-check">
+                  <label class="form-check-label" data-toggle="tooltip"
+                         title="Start grading at zero and add points">
+                    <input class="form-check-input" name="starting_points" type="radio" value="0"
+                           <% if ((rubric?.starting_points ?? 0) == 0) { %>checked<% } %>>
+                    Positive grading
+                  </label>
+                </div>
+                <div class="form-check">
+                  <label class="form-check-label" data-toggle="tooltip"
+                         title="Start grading at 100% (<%= max_points %> points) and subtract penalties">
+                    <input class="form-check-input js-negative-grading" name="starting_points" type="radio" value="<%= max_points %>"
+                           <% if (rubric?.starting_points == max_points) { %>checked<% } %>>
+                    Negative grading
```

Comment:
For the manual rubric (on an auto-graded question) I can't seem to save a rubric as "positive". I select "Positive grading" and click "Confirm" but then when I open the dialog again it has reset to "Negative grading".

Reply:
> For the manual rubric (on an auto-graded question) I can't seem to save a rubric as "positive". I select "Positive grading" and click "Confirm" but then when I open the dialog again it has reset to "Negative grading".

I can see how this may be triggered. In particular, if the question has 0 points for that type (manual or auto), then there is no real difference between positive or negative grading, since in both cases the points start at 0 (which is what this input controls).

Maybe I should hide the radios in case the maximum is already zero, to avoid confusion, and only allow a custom starting point on a checkbox? Or have a combined radio that has better information about this case?

Reply:
This is made clearer in d909578